app/models/request.js

```diff
@@ -66,12 +66,21 @@ export default Model.extend({
           }})
           .then(response => ({messages: response.messages}))
       });
+    }
+  }),
+
+  messages: computed('messagesRequest', function () {
+    let response = this.get('messagesRequest');
+    if (response) {
+      return response.get('messages');
     } else {
       return ObjectPromiseProxy.create({
         promise: EmberPromise.resolve([])
       });
     }
   }),
 
-  messages: alias('messagesRequest.messages'),
+  hasMessages: computed('messagesRequest', function () {
+    return this.get('messagesRequest').get('messages') != undefined;
+  }),
```

Comment:
It looks like this could be simplified with [`notEmpty`](https://api.emberjs.com/ember/3.13/functions/@ember%2Fobject%2Fcomputed/notEmpty) macro:
```js
hasMessages: notEmpty('messagesRequest.messages'),
```

Reply:
hmmmm ... it might be a macro (not sure which one), but this is meant to catch the case when `request` does not have a `messages` key ... i.e. when the feature is off, and we do not want to display the whole "build config validation" section, including it's header.

Reply:
Ah, I missed that point! Thanks for clarification 😄 It makes a lot of sense then 👍 Then I'd just suggest to combine `get`s:
```js
return this.get('messagesRequest.messages') != undefined;
```
It's essentially identical, but shorter :)